test/extended/builds/build_timing.go

```diff
@@ -90,6 +92,8 @@ var _ = g.Describe("[Feature:Builds][timing] capture build stages and durations"
 			g.By("starting the test docker build")
 			br, _ := exutil.StartBuildAndWait(oc, "test", "--from-file", dockerBuildDockerfile)
 			br.AssertSuccess()
+			// Bug 1716697 - ensure push spec doesn't include tag, only SHA
+			o.Expect(br.Logs()).To(o.MatchRegexp(`pushed image-registry\.openshift-image-registry\.svc:5000/.*/test@sha256:`))
```

Comment:
is it possible that if an error was introduced where both a tag followed by sha were in the spec, the tag would fall under the `*` in your regexp above

Reply:
According to [the distribution sources](https://github.com/docker/distribution/blob/master/reference/regexp.go#L37), a valid tag value can't include `/`, and the `.*` would have to match the `:` separator for a value that included a tag to be matched, so I think it's alright.

Reply:
thanks